### ISSUE TITLE
Text Swatch showing extra input field

### ISSUE THREAD

**Issue by rbur0425** [magento/magento2#11387]
### Preconditions

1. Magento 2.2 GA
2. PHP 7.0
3. MySQL 5.7

### Steps to reproduce

1. upgrade to magento 2.2 GA

### Expected result

1. Text swatches should be only inputs displayed

![text swatch should look like](https://user-images.githubusercontent.com/14355895/31464212-ceefa5f6-ae9e-11e7-87a6-5633ff23d178.jpeg)

### Actual result

1. Extra input fields under text swatches

![text swatch](https://user-images.githubusercontent.com/14355895/31464227-d8bca4bc-ae9e-11e7-8616-298bf1d6b3e5.jpg)

**Comment by magento-engcom-team** [comment]
@rbur0425 , thank you for your report.
We were not able to reproduce this issue by following the steps you provided. Please provide more detailed steps to reproduce or try to reproduce this issue on a clean installation or latest release.

**Comment by magento-engcom-team** [comment]
@rbur0425, we are closing this issue due to inactivity. If you'd like to update it, please reopen the issue.

**Comment by hstothard** [comment]
I also had the same issue. For me it was because until one of the more recent upgrades these fields were hidden by JS. The hiding of these fields is now in the below css file:

vendor/magento/module-swatches/view/frontend/web/css/swatches.css

Which is included into the page by:
vendor/magento/module-swatches/view/frontend/layout/catalog_cateogry_view.xml

My theme was not referencing this file and therefore the css was missing. Hope that helps anyone else who gets the same problem.

**Comment by davidedmonson** [comment]
I too had this issue. I was able to add .swatch-input{display:none} to my css and that fixed it for me. However, I do plan to use @hstothard fix in a future update.

**Comment by jmtakahashi** [comment]
@hstothard I'm currently on 2.1.10 and the file vendor/magento/module-swatches/view/frontend/web/css/swatches.css doesn't seem to contain the CSS to hide the extra input field.  I'm confused as to why this issue was closed.  Can you clarify where you found the css code?  Sorry to sound redundant, but it seems to be missing in my core files, which would mean this issue shouldn't be closed.

**Comment by hstothard** [comment]
Hi @jmtakahashi I am not sure with regards to 2.1.10. I am using version 2.2. It was added as inline styles with JS for me prior to upgrading. The css i was missing from swatches.css was:

.swatch-input {
    left: -1000px;
    position: absolute;
    visibility: hidden;
}

but display none also works well as per @davidedmonson

**Comment by jmtakahashi** [comment]
@hstothard thanks for clarifying.  I found the rule in the my core files and also found the reference in  the xml files under vendor/magento/module-swatches/view/frontend/layout/ 

`<css src="Magento_Swatches::css/swatches.css"/>`

I wonder why it's still showing on my store.  I also temporarily used custom css to hide it, but I wish it would work out of the box.

EDIT: @hstothard sorry about the confusion, after digging deeper, it seems there was a problem with my theme.  I just reuploaded newer files and the issue is gone.  Thanks, and apologies.
